node/utils/simulation.ts

```diff
@@ -41,13 +43,23 @@ const getMarketingData = (segment?: SegmentData) => {
 export const getSimulationPayloadsByItem = (
   item: ItemWithSimulationInput,
   segment?: SegmentData,
-  regionId?: string
+  regionId?: string,
+  useSellerFromRegion?: Boolean
 ) => {
+  let sellerSelected = '1'
+  if (useSellerFromRegion) {
+    const regionV1 = isBase64(regionId)
+    sellerSelected = regionV1 && regionId ? atob(regionId) : '1'
```

Comment:
Should this be really '1'?
I mean today this can be null? (from what I remember this is actually what happens in many times). 
Also shouldn't it be the same logic for regionV2?
Or this feature will be blocked for V2 region on checkout?

Reply:
Done!